src/styles/pages/login.css

```diff
@@ -76,3 +76,14 @@
   color: var(--green-light-color);
   text-decoration-line: underline;
 }
+
+.error-message {
+  visibility: hidden;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.active {
+  visibility: visible;
+  color: var(--red-light-color);
```

Comment:
Why don't set it in .error-message? Btw, if I add .active for any element, it will apply this style? If yes, pls make sure it works only for .error-message